lib/hydranorth/preservation_queue.rb

```diff
@@ -0,0 +1,17 @@
+module Hydranorth::PreservationQueue
+
+  QUEUE_NAME = YAML.load(File.read('config/preservation.yml'))[Rails.env]['queue_name']
```

Comment:
Then you can just use: `Rails.application.secrets.preservation.queue_name` or what have you